src/test/librbd/CMakeLists.txt

```diff
@@ -123,6 +123,7 @@ target_link_libraries(ceph_test_librbd_api
 set_target_properties(ceph_test_librbd_api PROPERTIES COMPILE_FLAGS
   ${UNITTEST_CXX_FLAGS})
 
+ifdef(LINUX)
```

Comment:
@tchaikov 
Arggh, sometimes my head overflows with all the syntax versions.
And I did not copy my wip-version because master changed too much.
And redoing it seemed easier.
have fixed/rebased.